configmap/mariadb/start/start.sh

```diff
@@ -19,6 +19,6 @@ fi
 echo "-- Start mariadb server."
 mysqld
 if [ $? -ne 0 ]; then
-    >&2 echo "ERROR: failed to start the replication database"
+    >&2 echo "ERROR: failed to start the Qserv database"
```

Comment:
this error message does not appear to match the echo'ed statement on line 19.

Reply:
:+1: